Cisco-Talos/clamav
# WIP: add OpenRC service scripts

As promised, here are four upstream service scripts for clamav-miler, clamd, clamonacc, and freshclam that make use of the new root-owned PID files.

To test this out, I've been running

```
$ git clean -x -f -d
$ ./autogen.sh
$ ./configure --disable-clamdtop --enable-milter --enable-clamonacc --enable-openrc --prefix=/home/mjo/tmp/clamav
$ make runstatedir=/home/mjo/tmp/clamav/run
$ make install
```

but of course you'll want to replace `/home/mjo/tmp/clamav` with a path of your own. This needs some more testing, but I believe everything kinda works the way it should. Feedback would be most welcome at this point!
